sound/mpd/Makefile

```diff
@@ -101,104 +98,90 @@ define Package/mpd-avahi-service/conffiles
 /etc/avahi/services/mpd.service
 endef
 
-TARGET_CFLAGS += -ggdb3
-TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib $(if $(ICONV_FULL),-liconv)
-EXTRA_CXXFLAGS += $(if $(CONFIG_GCC_VERSION_4_8),-std=gnu++11,-std=gnu++14)
```

Comment:
In all honesty I'm very sure that you'll see breakage pretty much everywhere in the tree. The EXTRA_CXXFLAGS stuff should be enough?